src/components/WebexMemberRoster/WebexMemberRoster.scss

```diff
@@ -23,4 +26,8 @@
     line-height: 1.5rem;
     color: var(--wxc-text-color);
   }
+
+  .members {
+    overflow-y: auto;
```

Comment:
+ flex: 1 so that it occupies all available space. Also, please check if the scrollbar appears if there are more members than the available space, if not, try min-height: 0